requirements.txt

```diff
@@ -0,0 +1,4 @@
+mkdocs<=0.16
```

Comment:
@pelson why are you stick with `<=0.16`?

BTW we are not married to `mkdocs`, it was convenient at the time but I'm happy to move to anything you find easier to maintain.

Reply:
I'm easy. Just trying to document the current state of play.

mkdocs moved forwards and have a different config setup at this point. Should be easy to fix, but as I say, looking to simply document what is here already first 😄

Reply:
Thanks! I'll look into updating the config setup later today.

Reply:
@pelson I updated the config and more in https://github.com/ugrid-conventions/ugrid-conventions/pull/44

BTW, your changes here (the `md_toc github src/index.md -i`) made into the page automatically thanks to `doctr`.